index.bs

```diff
@@ -1943,53 +1947,88 @@ Constructors</h4>
         Sending a <a>control message</a> to start processing means
         executing the following steps:
     
-        <div class="addition proposed" id="c2400-5">
-        <span class="marker">Proposed Addition</span><br>
-            <a href="https://github.com/WebAudio/web-audio-api/issues/2400">Issue
-                2400</a> Access to a different output device
-        <div class="amendment-buttons"></div>
-        <del cite=#2400>
-            1. Attempt to <a href="#acquiring">acquire system resources</a>.
-                In case of failure, abort the following steps.
-
-            3. Set the {{[[rendering thread state]]}} to <code>running</code> on the {{AudioContext}}.
+        <div class="addition proposed">
+            <div class="addition proposed" id="c2400-5" style="border:0; margin:0; padding:0">
+                <span class="marker">Proposed Addition
+                    <a href="https://github.com/WebAudio/web-audio-api/issues/2400">Issue 2400</a>.
+                </span>
+                Access to a different output device
+                <div class="amendment-buttons"></div>
```

Comment:
do you need these? don't these get added by the fixup.js script?

Reply:
I've been following other existing examples of this spec. As far as I can tell, I don't see any reference to `fixup.js` in the spec source file. Perhaps I am missing something?

Reply:
Bikeshed adds fixup.js when generating the static html file. Just confirm that the purple boxes are all ok (and if possible, see if removing the div makes any difference).

Reply:
With this div, the button shows up:

![Screenshot 2024-05-31 at 8 40 25 PM](https://github.com/WebAudio/web-audio-api/assets/676891/af7429b1-bb28-4ea4-96be-49440ec4e049)


When I remove the div, the button disappears:

![Screenshot 2024-05-31 at 8 39 09 PM](https://github.com/WebAudio/web-audio-api/assets/676891/9b25a456-b234-4d84-bde4-793e410b5af9)